views/index.haml

```diff
@@ -3,7 +3,7 @@
   %head
     %title GitHub Preview
     %script{:type => "text/javascript", :src => "http://code.jquery.com/jquery-1.6.2.min.js"}
-    %script{:type => "text/javascript", :src => "http://documentcloud.github.com/underscore/underscore-min.js"}
+    %script{:type => "text/javascript", :src => "http://underscorejs.org/underscore-min.js"}
```

Comment:
Line is too long. [92/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace